config/locales/server.en.yml

```
@@ -1381,8 +1381,8 @@ en:
     facebook_app_secret: "App secret for Facebook authentication, registered at <a href='https://developers.facebook.com/apps/' target='_blank'>https://developers.facebook.com/apps</a>"
 
     enable_github_logins: "Enable Github authentication, requires github_client_id and github_client_secret. See <a href='https://meta.discourse.org/t/13745' target='_blank'>Configuring GitHub login for Discourse</a>."
-    github_client_id: "Client id for Github authentication, registered at <a href='https://github.com/settings/applications/' target='_blank'>https://github.com/settings/applications</a>"
-    github_client_secret: "Client secret for Github authentication, registered at <a href='https://github.com/settings/applications/' target='_blank'>https://github.com/settings/applications</a>"
+    github_client_id: "Client id for Github authentication, registered at <a href='https://github.com/settings/developers/' target='_blank'>https://github.com/settings/developers</a>"
+    github_client_secret: "Client secret for Github authentication, registered at <a href='https://github.com/settings/applications/' target='_blank'>https://github.com/settings/developers</a>"
```

One more change needed here ;)
```suggestion
    github_client_secret: "Client secret for Github authentication, registered at <a href='https://github.com/settings/developers/' target='_blank'>https://github.com/settings/developers</a>"
```

---

Whoops, thanks for catching that. :blush: This new GitHub suggestion feature is pretty cool though. :-)